org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/ParameterizedExpressionsTestLanguageGrammarAccess.java

```diff
@@ -31,7 +31,7 @@
 		private final RuleCall cLabelledStatementParserRuleCall_2 = (RuleCall)cAlternatives.eContents().get(2);
 		private final RuleCall cExpressionStatementParserRuleCall_3 = (RuleCall)cAlternatives.eContents().get(3);
 		
-		//RootStatement <Yield Statement:
+		//RootStatement <YieldStatement:
```

Comment:
There is something missing here: I think it should read `RootStatement<Yield> returns Statement:`